microsoft-edge/devtools-guide-chromium/resources/index.md

```diff
@@ -20,145 +20,181 @@ ms.date: 05/04/2021
 
-To view the resources of a webpage organized by directory:
+1. To open DevTools, right-click the webpage, and then select **Inspect**.  Or, press `Ctrl`+`Shift`+`I` (Windows, Linux) or `Command`+`Option`+`I` (macOS).  DevTools opens.
 
-1. Open DevTools.
+1. In DevTools, on the main toolbar, select the **Sources** tab.  If that tab isn't visible, click the **More tabs** (![More tabs icon](../media/more-tabs-icon-light-theme.png)) button.
 
-1. Select the **Sources** tool, and then in the **Navigator** pane in the upper left, select the **Page** tab.
+1. In the **Navigator** pane in the upper left, select the **Page** tab.
 
-1. Click the **More options** (...) button to the right of the **Page** tab, and then select **Group by folder**.
+1. By default, resource files are grouped by folder.  If the resources are all listed alphabetically by file name instead, click the **More options** (...) button to the right of the **Page** tab, and then select **Group by folder**:
 
-   ![The Page tab in the Navigator pane of the Sources tool.](../media/resources-sources-page-empty.msft.png)
+   ![The webpage's resource files grouped by folder in the Page tab of the Sources tool](../media/resources-sources-page-empty.msft.png)
 
-    Here is a breakdown of the non-obvious items in the previous figure.
+    The **Page** tab contains various node types, including:
 
     | Page item | Description |
-    |:--- |:--- |
-    | `top` | The main document [browsing context](https://developer.mozilla.org/docs/Web/HTML/Element/iframe). |
-    | `airhorner.com` | The domain.  All resources nested under it come from that domain.  For example, the full URL of the `comlink.global.js` file is probably `https://airhorner.com/scripts/comlink.global.js`. |
-    | `scripts` | A directory. |
+    |---|---|
+    | `top` | The main document _browsing context_, such as a browser tab, a browser window, or a frame. See [Browsing context](https://developer.mozilla.org/en-US/docs/Glossary/Browsing_context) or [\<iframe\>: The Inline Frame element](https://developer.mozilla.org/docs/Web/HTML/Element/iframe). |
+    | `microsoftedge.github.io` | The domain.  All resources nested under it come from that domain.  For example, the full URL of the `buttons.js` file is [https://microsoftedge.github.io/Demos/devtools-a11y-testing/buttons.js](https://microsoftedge.github.io/Demos/devtools-a11y-testing/buttons.js). |
+    | `css`, `photos` | Directories. |
     | `(index)` | The main HTML document. |
-    | `sw.js` | A service worker runtime context. |
+    | `buttons.js` | A script to handle button presses on the page. |
 
-1. Select a resource to view it in the **Editor**.
+1. Select a resource file to view it in the Editor of the **Sources** tool:
 
-   ![View a file in the Editor.](../media/resources-sources-page-resource.msft.png)
+   ![Viewing a resource file in the Editor of the Sources tool](../media/resources-sources-page-resource.msft.png)
```

Comment:
@dan-wesley i did hard refresh but am seeing no red boxes on this png's tabs, at rendered view:

https://github.com/MicrosoftDocs/edge-developer/pull/2339/files#r1052576441

Reply:
try again